### PR TITLE
Home page validation

### PR DESCRIPTION
**Pull request by walahamad**
This PR handle the following tasks:
* Navigation menu existence and open / close (SMK-4).
* Positive form filling (Filling the login form) (SMK-16).

### CODE REVIEW

File: src/com/generic/selector/HomePageSelectors.java [hunk 1]
```diff
@@ -28,4 +28,20 @@
 	public static final String footerLogo = "footer-ocm-logo";
 	public static final String footerEmailSignUp = "footer-email-sign-up";
 
+	// Header navigation menu selectors.
+	public static final cselector navigationIcon = new cselector("","css,.nav-toggle", "css,.t-header__menu-icon .pw-button");
+	public static final cselector menuItems = new cselector("css,#css-top-navigation #flyout>.menuItem>a","css,#gwt-top-navigation .active>.gwt-nav-item-has-submenu>.gwt-HTML", "css,.navigation-modal_class .pw-link");
+	public static final cselector selectedMenuHeader = new cselector("", "css,#gwt-top-navigation .active>.gwt-nav-item-has-submenu.opened>.gwt-HTML", "css,.m-navigation__fades-in .pw-header-bar .c-sheet__title");
```
Comment (ibatta) [inline src/com/generic/selector/HomePageSelectors.java]: Are those mobile / ipda specific  ?

File: src/com/generic/util/SelectorUtil.java [hunk 2]
```diff
@@ -418,7 +418,16 @@ public WebElement apply(WebDriver driver) {
 							if (value.contains("pressEnter")) {
 								tempVal = value.split(",")[0];
 							}
-							field.sendKeys(tempVal);
+							if (browser.contains("mobile"))
```
Comment (ibatta) [inline src/com/generic/util/SelectorUtil.java]: good job/ please just use the global variables browsers

File: src/com/generic/util/SelectorUtil.java [hunk 3]
```diff
@@ -860,4 +869,12 @@ public static WebElement getelement(String selector) throws Exception {
...
 
 	}
-}
\ No newline at end of file
+
+	public static String getCurrentPageUrl() throws Exception {
+		return SelTestCase.getDriver().getCurrentUrl();
+	}
+
+	public static void clickOnWebElement(WebElement element) throws Exception {
```
Comment (ibatta) [inline src/com/generic/util/SelectorUtil.java]: @walaahamad why we need this ?

File: src/com/generic/selector/HomePageSelectors.java (hunk 1 above)
Comment (walahamad) [reply src/com/generic/selector/HomePageSelectors.java]: desktop, ipad and mobile.

File: src/com/generic/util/SelectorUtil.java (hunk 3 above)
Comment (walahamad) [reply src/com/generic/util/SelectorUtil.java]: To click on item from the listArray webElement.

**Pull request merged**